appveyor.yml

```diff
@@ -19,18 +17,16 @@ cache:
 
 before_build:
   - nuget sources add -name bddhelper -source https://ci.appveyor.com/nuget/ospsuite-bddhelper 
-  - nuget install packages.config -OutputDirectory packages # cannot use automatic artifact publish because it's not recognized in a C++/CLI project
-  - ps: (get-content src\OSPSuite.FuncParser\version.h) | foreach-object {$_ -replace "RELEASE 0", "RELEASE $env:RELEASE" -replace "MAJOR 0", "MAJOR $env:MAJOR" -replace "MINOR 0", "MINOR $env:MINOR" -replace "BUILD 0", "BUILD $env:APPVEYOR_BUILD_NUMBER"} | set-content src\OSPSuite.FuncParser\version.h
-
+  - ps: (get-content src\OSPSuite.FuncParserNative\version.h) | foreach-object {$_ -replace "RELEASE 0", "RELEASE $env:RELEASE" -replace "MAJOR 0", "MAJOR $env:MAJOR" -replace "MINOR 0", "MINOR $env:MINOR" -replace "BUILD 0", "BUILD $env:APPVEYOR_BUILD_NUMBER"} | set-content src\OSPSuite.FuncParser\version.h
+  - dotnet restore
 
 after_build:
   - cmd: nuget pack src\OSPSuite.FuncParser\OSPSuite.FuncParser.nuspec -version %app_version%
   - ps: Get-ChildItem .\OSPSuite.FuncParser.*.nupkg | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
 
 test:
   assemblies:
-  - '**\*.Tests.Release.Win32.dll'
-  - '**\*.Tests.Release.x64.dll'
+  - '**\*.Tests.dll'
```

Comment:
not sure if the tests on AppVeyour do not start due to the wrong test dll location.
It's stored under `$SolutionDir\tests\OSPSuite.FuncParser.Tests\bin\Release\netcoreapp2.1\OSPSuite.FuncParser.Tests.dll`